cmd/zoekt-mirror-gerrit/main.go

```diff
@@ -166,6 +166,7 @@ func main() {
 			"zoekt.name":           name,
 			"zoekt.gerrit-project": k,
 			"zoekt.gerrit-host":    rootURL.String(),
+			"zoekt.archived":       marshalBool(v.State != "ACTIVE"),
```

Comment:
Note: Gerrit repos can have three states: ACTIVE, READ_ONLY and HIDDEN. Therefore both READ_ONLY _and_ HIDDEN repositories are considered archived in this code.

We may want to refine that down to only consider READ_ONLY repositories archived. I'm not familiar with Gerrit, so I'm not sure what the best option is.

Reply:
Hidden projects can still be edited. From reading the docs the closest thing to github's archive is just the READ_ONLY state.

Reply:
I've changed this to just check for READ_ONLY as suggested.